Source/Extensions/Blazorise.Video/Video.razor.cs

```diff
@@ -70,37 +70,41 @@ protected override async Task OnAfterRenderAsync( bool firstRender )
     {
         if ( firstRender )
         {
-            await JSModule.Initialize( DotNetObjectRef, ElementRef, ElementId, new
+            await JSModule.Initialize( DotNetObjectRef, ElementRef, ElementId, new VideoJSOptions
             {
-                Controls,
-                ControlsList,
-                SettingsList,
-                AutomaticallyHideControls,
-                AutoPlay,
-                AutoPause,
-                Muted,
-                Source,
-                Poster,
-                Thumbnails,
+                Controls = Controls,
+                ControlsList = ControlsList,
+                SettingsList = SettingsList,
+                AutomaticallyHideControls = AutomaticallyHideControls,
+                AutoPlay = AutoPlay,
+                AutoPause = AutoPause,
+                Muted = Muted,
+                Source = Source,
+                Poster = Poster,
+                Thumbnails = Thumbnails,
```

Comment:
If we want to, we can make a dedicated constructor, so it's more close to the previous implementation and also enforces the number of arguments.

I do like seeing the properties explicitly defined however.